utils.py

```diff
@@ -0,0 +1,9 @@
+import sublime
+
+def get_setting(view_or_window, setting_name, default_value):
+    if view_or_window is not None:
+        local_settings = view_or_window.settings().get("PrettierdFormat", {})
+        if setting_name in local_settings:
+            return local_settings[setting_name]
+
+    return sublime.load_settings("prettierd_format.sublime-settings").get(setting_name, default_value)
```

Comment:
This looks wasteful in loading the settings file each time, but the [API docs](https://www.sublimetext.com/docs/api_reference.html#sublime.load_settings) say this doesn't actually reload anything; it is all cached under the hood.